src/testing/mockUseLoadFromEndpoint.ts

```diff
@@ -0,0 +1,41 @@
+import useLoadFromEndpoint from '../hooks/useLoadFromEndpoint'
```

Comment:
Just copied from Integreat, right? Normally I'd say you should use arrow functions but if it is just copied nvm :D

Reply:
You're right I'll change it to arrow functions for consistency